tsconfig.json

```diff
@@ -5,6 +5,7 @@
     "jsx": "preserve",
     "sourceMap": true,
     "strict": true,
+    "skipLibCheck": true,
```

Comment:
79cfcfb commit addresses previous failure. Please consider https://www.typescriptlang.org/tsconfig#skipLibCheck during review.

Reply:
Which previous failures is this flag addressing? Is it the build failures where multiple copies of `@types/react` is the cause?

Reply:
Please note the commit I referenced with linked error info. This change resolves for `@types/download` duplicates scattered throughout various scripting tools. Note that the recommended `skipLibCheck` setting does not skip type checking for component dependency references. I think it is not only safe, but likely a preferred change for Garden.